src/window_actorstatus.cpp

```diff
@@ -45,24 +45,26 @@ void Window_ActorStatus::DrawStatus(){
 
 	// Draw Hp
 	contents->TextDraw(1, 2, 1, lcf::Data::terms.health_points);
-	DrawMinMax(90,2,actor->GetHp(), actor->GetMaxHp());
+	int color = (actor->GetHp() == 0 ? Font::ColorKnockout : (actor->GetHp() <= actor->GetMaxHp() / 4 ? Font::ColorCritical : Font::ColorDefault));
+	DrawMinMax(90,2,actor->GetHp(), actor->GetMaxHp(), color);
```

Comment:
Can we fix up the spacing between `DrawMinMax` function arguments?
I know it has been in there before, however it looks ugly.

Reply:
Fixed.